bokeh/model.py

```diff
@@ -739,9 +723,22 @@ def __exit__(self, type, value, traceback):
         for model in self._to_remove_after:
             model.document.remove_root(model)
 
-    def __enter__(self):
+    def __enter__(self, tried_already=False):
         for model in self._to_remove_after:
-            self._doc.add_root(model)
+            try:
+                self._doc.add_root(model)
+            except:
```

Comment:
We don't allow bare `except` statements. Please be specific on what exactly you want to catch here.